syntax/std_test.go

```diff
@@ -9,6 +9,15 @@ func TestStdTuple(t *testing.T) {
 	AssertCodesEvalToSameValue(t, `(a:1)`, `//tuple({"a":1})`)
 	AssertCodesEvalToSameValue(t, `(a:1, b:2)`, `//tuple({"a":1, "b":2})`)
 
+	AssertCodesEvalToSameValue(t, `('':1)`, `//tuple({"":1})`)
+	AssertCodesEvalToSameValue(t, `('':1)`, `//tuple({{}:1})`)
+	AssertCodesEvalToSameValue(t, `('':1)`, `//tuple({[]:1})`)
+	AssertCodesEvalToSameValue(t, `('0': 0)`, `//tuple({0: 0})`)
+	AssertCodesEvalToSameValue(t, `('()':())`, `//tuple({(): ()})`)
```

Comment:
//tuple should fall for non-string keys.

Reply:
*fail